test/src/post-make.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+function main() {
+	for use_dir in "${@}"; do
+		# we only want to build libinconsistent.dylib as a fat mach-o
+		if [[ "${use_dir}" == fat* ]]; then
+			inconsistent_for "${use_dir}"
+		fi
+	done
+}
+
+function inconsistent_for() {
+	# this mess just splits a fat directory spec like fat-i386-x86_64 into
+	# its constituent arch(3) pairs (e.g., i386 and x86_64)
+	local fat_dir="${1}"
+	local split_fat_dir=(${fat_dir//-/ })
```

Comment:
I was being mostly sarcastic (since I always end up looking up the syntax for string splitting/truncation in bash), but the second alternative looks great to me :smile: